### ISSUE TITLE
problem: people don't always update data

### ISSUE THREAD

**Issue by gazhayes** [howlongistheline/howlongistheline.org#173]
problem: people don't always update data

solution: use Google's popular times API somewhere (if it's free, haven't checked yet just noting this down).
https://github.com/m-wrzr/populartimes/ looks like a nice library for it.

**Comment by jimkleine** [comment]
@gazhayes If I understand correctly, it is chargeable and there is [no free tier](https://developers.google.com/places/web-service/usage-and-billing#find-place).

The feature still doesn't seem to be available as an official Google API.  There is an [open Google request](https://issuetracker.google.com/issues/35827350) (login required to view), it was assigned to a Google product manager in 2018, but it is currently being blocked by another issue that seems to be only visible to Google staff.

The data visible on Google Maps (it **is** available for the local supermarkets I sampled) doesn't appear to be absolute numbers: it is a ratio of the average occupancy of the store, which suggests that it may be difficult into translate to the kind of information we need.
